### DISCUSSION TITLE
Getting Error while implementing OR Conditon

### DISCUSSION THREAD

**Discussion by rpulekar** [NRules/NRules#351] [Q&A] [answered]
Getting error :  Rule must have at least one match action 

I need to Add OR condition in my Rule some thiing like below 

When()
         .Or(x=>x.Match( () => input , input.condition = 'value1' )
         .Or(x=>x.Match( () => input , input.condition = 'value2' )
));

this is not working , can you please help me to construct the rule

**Comment by snikolayev** [comment, marked as answer]
@rpulekar it would be nice when you say "it's not working" to give not just a snippet, but a unit test that can be executed, to demonstrate what exactly is not working, and also include the error message/trace/etc.

Just at a glance, what you are looking for is this:
```c#
When()
    .Match(() => input , input.condition == 'value1' || input.condition == 'value2');
```
